packages/components/src/config/motion.ts

```diff
@@ -0,0 +1,25 @@
+// todo: name of this file?
+// todo: types?
```

Comment:
I had a plan for the future to work on animations and establish proper rules and patterns, as well as make the app more animated. The second part might not happen anytime soon though :(

Reply:
cool :+1: for now I just need some share-ability